backend/btrixcloud/colls.py

```diff
@@ -65,6 +65,7 @@ async def add_collection(
         name: str,
         crawl_ids: Optional[List[str]],
         description: Optional[str] = None,
+        is_public: Optional[bool] = False,
```

Comment:
See other comment, we can reduce this to just `new_coll: CollIn` and then set the values in the `Collection` init accordingly